ivy/functional/backends/numpy/linear_algebra.py

```diff
@@ -340,7 +340,7 @@ def trace(
     axis2: int = 1,
     out: Optional[np.ndarray] = None,
 ) -> np.ndarray:
-    return np.trace(x, offset=offset, axis1=axis1, axis2=axis2, out=out)
+    return np.trace(x, offset=offset, axis1=axis1, axis2=axis2)
```

Comment:
ideally we should be using the `out` argument as it's natively supported in `np.trace`. Are there any issues being caused due to the `out` argument?

Reply:
Sorry, I'm not sure why I removed that out argument. I added it now, and all the tests are passing.👍🏻

Reply:
awesome, thanks!